lldb/source/Commands/CommandObjectTarget.cpp

```diff
@@ -1593,6 +1595,7 @@ static uint32_t LookupSymbolInModule(CommandInterpreter &interpreter,
     return 0;
 
   SymbolContext sc;
+  bool use_color = interpreter.GetDebugger().GetUseColor();
```

Comment:
Unlike in llvm we do `const` variables (and there's examples of that in the surrounding code) so let's make this `const bool use_color`.

Reply:
Done.